src/store-api/src/mito_engine_options.rs

```diff
@@ -59,6 +59,8 @@ pub const MEMTABLE_PARTITION_TREE_DATA_FREEZE_THRESHOLD: &str =
 /// Option key for memtable partition tree fork dictionary bytes.
 pub const MEMTABLE_PARTITION_TREE_FORK_DICTIONARY_BYTES: &str =
     "memtable.partition_tree.fork_dictionary_bytes";
+/// Option key for skipping WAL.
+pub const SKIP_WAL_KEY: &str = "skip_wal";
```

Comment:
Maybe we can add a comment to remind us that we should check whether the metric engine metadata region has to remove the option.